esp/scm/ws_dfu.ecm

```diff
@@ -812,6 +821,30 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPrequest [nil_remove] DFUCreateAndPublishRequest
+{
+    string LogicalName;
+    string Cluster;
+    string JobDescription;
+    string ECLRecordDefinition;
+    
+    bool ReturnJsonTypeInfo(false);
+    bool ReturnBinTypeInfo(false);
+    ESPenum SecAccessType AccessType;
+    int AccessTokenExpirationMinutes(1);
+};
+
+ESPresponse [exceptions_inline, nil_remove] DFUCreateAndPublishResponse
+{
+    string LogicalName; //Copy from request
+    string UserID; //the uesr of the DFUCreateAndPublishRequest
+    string JobDescription; //Copy from request
+    ESPenum SecAccessType AccessType; //Copy from request
+    int AccessTokenExpirationMinutes; //Copy from request if < 24hrs. Max 24hrs
+    ESPstruct DFUFileDetail FileDetail;
+    string AccessToken;
```

Comment:
@jakesmith i think expressed an issue w/ this name, please provide an accurate alternative.

Reply:
Also, this security artifact will follow @jakesmith new format